src/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed/Configuration/EnvironmentConfigurationHelper.cs

```diff
@@ -75,6 +75,18 @@ private static TracerProviderBuilder SetExporter(this TracerProviderBuilder buil
                         options.ExportProcessorType = ExportProcessorType.Simple; // for PoC
                     });
                     break;
+                case "OTLP":
```

Comment:
```suggestion
                case "otlp":
```

from https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#exporter-selection

Reply:
@dszmigielski remove the case for "OTLP"